### DISCUSSION TITLE
links broken on available databases page / meaning of numbers after CATH IDs

### DISCUSSION THREAD

**Discussion by EDohmen** [flatironinstitute/deepblast#158] [Q&A] [answered]
Hi tmvec and deepblast team and thanks for the models and tools you develop!
I'm currently testing with a student if we can use tmvec and deepblast for some project and we were wondering what the last numbers after the CATH IDs (I guess from the metadata) in the column "database_id" mean. 
```
query_id        rank    database_id     tm-score
CCT71297_1      0       cath|4_3_0|1j1eF00/40-191       0.6801977
CCT71297_1      1       cath|4_3_0|1j1eC01/35-148       0.6698675
```
Are these aa positions? And if so, are these positions in the database protein sequences or rather already where it matched in the query sequence (then it would probably not be available without the alignment step already, right)?

I tried to have a look at the metadata files here https://github.com/flatironinstitute/deepblast/wiki/Available-databases but all the link targets there can't be found and lead as it seems to a local path (that's why I'm not posting here the path/link itself, since it contains a home folder name etc. and you may not want to have that visible in the public repo).

Again, thanks for your work and we would appreciate any help and hints about correct interpretation of the tmvec output!
Best,
Elias

**Comment by mortonjt** [comment]
Hi, thanks for raising this issue.  I just ran the wget commands and I was able to download everything.
Can you confirm that you were able to run wget?

**Reply by EDohmen** [reply]
Oh yes, thanks, I can verify. The wget command actually points to a different target (the hyperlinks [download "here"] contain /mnt/home/ in between instead of flatironinstitute.org), which I didn't realise and I was able to download the files with wget with the correct links below.
Cheers!

**Comment by EDohmen** [comment]
Should I ask the question about the tm-vec output somewhere else or is it okay in here, even if it is linked to deepblast? I didn't find something similar (Q&A) for tm-vec itself.

**Reply by mortonjt** [reply]
Hi yes feel free to post tmvec questions here as well

**Reply by EDohmen** [reply]
Thanks, but I was just not very clear I realise now. I meant actually the first question from my original post about the tmvec output and what the last numbers from the database_id mean, or how to interpret these correctly, respectively. Can anyone explain this or point to documentation where it's described?
Thanks a lot again!

**Comment by mortonjt** [comment]
Got it. I’m looking over the code, the database_id is the positional index
of the database. So 0 would correspond to the first sequence in the
database, 1 would correspond to the second, etc.

We may want to output the raw id / sequence in future iterations

**Reply by EDohmen** [reply]
Thanks for the answer, but I meant for example the "40-191" from the example output in the first post (the string in the field database_id also shown below) running tmvec-search against the cath db you provide
```
cath|4_3_0|1j1eF00/40-191
```
Are the 40 and 191 a range of amino acid positions in the sequence belonging to the cath ID "1j1eF00" or are these numbers something different?

**Reply by EDohmen** [reply, marked as answer]
Oh, now I found it myself. It comes indeed from the CATH DB directly (part of the metadata) and represents the residues extracted from the underlying PDB files. 
Sorry for the confusion and thanks for your answers - also having the sequence in future iterations would be interesting/useful, so this info from your last post (that it might be implemented in the future) is good to know!
Thank you for the replies here and the tool development again!
